app/assets/stylesheets/sephora_style_guide/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,39 @@
+.breadcrumb {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  font-size: 11px;
+  height: 11px;
+  padding: 0;
+}
+
+.breadcrumb-item {
+  display: flex;
+  line-height: 13px;
+
+  &::before {
+    content: "/";
+    display: inline-block;
+    color: $grey-dark;
+    padding: 0 3px;
```

Comment:
When not needed, try not to overwrite CSS just for convenience. So if you don't need the 0 padding on top and bottom, this should be written as:
```css
padding-left: 3px;
padding-right: 3px;
```